react/components/ProductImages/index.js

```diff
@@ -109,6 +110,21 @@ const ProductImages = ({
       </div>
     )
 
+  else if (displayMode === DISPLAY_MODE.FIRST_IMAGE)
```

Comment:
Add keys to the if clauses and check if there's any image before rendering the `[0]` position.